app/controllers/houses_controller.rb

```diff
@@ -1,11 +1,11 @@
-class HouseController < ApplicationController
+class HousesController < ApplicationController
 
   def index
-    @houses = Houses.all
+    @houses = House.all
   end
 
   def show
-    @houses = Houses.all
+    @houses = House.all
```

Comment:
The purpose of `show` is to display information about a single House. But `House.all` returns all of the houses in the database...